### ISSUE TITLE
Cannot read property 'sourceHashes' of undefined when clicking some item image on organizer

### ISSUE THREAD

**Issue by duckmanBR** [DestinyItemManager/DIM#5363]
**What version of DIM are you using?**
6.16.0.7363 Beta

**What Browser and OS are you using?**
Chrome 83.0.4103.106

**Describe the bug and how to make it happen**
Open Organizer, filter armor and warlock (don't have enough items on Hunter/Titan to check there).  Pick any slot.
Click the item icon (to open the inventory popup).  On Scatterhorn and Solstice items it will return a "Something went wrong / Cannot read property 'sourceHashes' of undefined"

**Include devtools console logs**
If you can, open the devtools console (ctrl+shift+J on PC and cmd+option+J on Mac) and paste what you see in the window that pops up:

```
{destinyVersion: 2, location: {…}, bucket: {…}, hash: 2546370410, type: "Helmet", …}
instrument.ts:129 TypeError: Cannot read property 'sourceHashes' of undefined
    at f (PowerCapDisclaimer.tsx:24)
    at Gi (react-dom.production.min.js:153)
    at hs (react-dom.production.min.js:261)
    at sc (react-dom.production.min.js:246)
    at ic (react-dom.production.min.js:246)
    at Xs (react-dom.production.min.js:239)
    at react-dom.production.min.js:123
    at scheduler.production.min.js:19
    at Vo (react-dom.production.min.js:122)
    at Wo (react-dom.production.min.js:123)
    at Uo (react-dom.production.min.js:122)
    at A (react-dom.production.min.js:287)
    at $t (react-dom.production.min.js:68)
    at HTMLDocument.r (helpers.ts:85)
```

**Comment by webalienz** [comment]
This just started happening to me today.  I then went over to beta.destinyitemmanager.com and Organizer worked fine, until I did the following:

1. Select Armor
2. Select Any Class
3. Select Any Armor Slot
4. Results were shown.
5. I switched tabs back over to another page (in my case, it was Discord)
6. When I switched back over to the DIM tab, the results were gone and it was asking me to choose Weapon or Armor.
7. Repeated steps #1 through #3, then received the error message as OP.  

Now in both app and beta, I get the same error messages when trying to use Organizer for Armor.

In Firefox 77.0.1, the error message that shows up is 'c.a.raid is undefined'.  I'm on the same version of Chrome as OP.

This all started sometime after 8:45:08 AM, when I was using it.  My DIM Sync history shows that I changed an item to "Infuse" at that time, and at some point I went back to Organize some more when I noticed the error occurring.
